prov/efa/src/rxr/rxr.h

```diff
@@ -83,9 +83,9 @@ static inline void rxr_poison_mem_region(uint32_t *ptr, size_t size)
 #endif
 
 /*
- * Set alignment to x86 cache line size.
+ * Set alignment to page size so that we can use MADV_DONTFORK.
  */
-#define RXR_BUF_POOL_ALIGNMENT	(64)
+#define RXR_BUF_POOL_ALIGNMENT	(4096)
```

Comment:
page size is not always 4096.

It is different on different os + cpu architecture.

libfabric has an utility function you can use to get page size `ofi_get_page_size`